modal/object.py

```diff
@@ -44,6 +44,7 @@ class _Object:
     _object_id: str
     _client: _Client
     _is_hydrated: bool
+    _is_rehydrating: bool
```

Comment:
You set `_is_rehydrated` (past tense) below; looks like you read the latter in the code too.